tests/help.rs

```diff
@@ -347,6 +347,16 @@ FLAGS:
     -H, --help       Print help information
     -v, --version    Print version information";
 
+static HELP_CONFLICT: &str = "conflict 
+
+USAGE:
+    conflict [FLAGS]
+
+FLAGS:
+        --help       Prints help information
+    -h               
```

Comment:
This looks weird

Reply:
Yeah, but I think this is intentional. We override the generated options when we specify an option.